crates/rspack_core/src/chunk_graph/chunk_graph_module.rs

```diff
@@ -203,9 +203,12 @@ impl ChunkGraph {
           )
         });
 
-      if let Some(module) = module_graph.module_by_identifier(&module.identifier()) {
-        let export_type = module.get_exports_type(strict);
-        export_type.dyn_hash(&mut hasher);
+      if let Some(strict) = strict {
+        if let Some(build_meta) = module.build_meta() {
+          strict.dyn_hash(&mut hasher);
+          build_meta.default_object.dyn_hash(&mut hasher);
+          build_meta.exports_type.dyn_hash(&mut hasher);
+        }
```

Comment:
There is no need to get exports type to generate hash. Use build meta and exports info directly.